classes/student.rb

```diff
@@ -1,13 +1,18 @@
 require_relative 'person'
-# student subclass
 
 class Student < Person
-  def initialize(classroom, _id, _age, _name = 'unkown', parent_permission: true)
-    super
-    @classroom = classroom
+  attr_accessor :classroom
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    super(age, name, parent_permission: parent_permission)
```

Comment:
- [ ] Please fix the error when creating a student.
![image](https://user-images.githubusercontent.com/38635158/170824431-c427ee71-ae15-46ba-b3a5-fdac270c77be.png)

_Next code reviewer please check if creating rentals or list the rentals. Right now ı can't create any person so ı can't create the rentals._